test_server.py

```diff
@@ -32,7 +32,34 @@ def test_get_product_list(self):
         resp = self.app.get('/products')
         #print 'resp_data: ' + resp.data
         self.assertTrue( resp.status_code == HTTP_200_OK )
+
+        data = json.loads(resp.data)
         self.assertTrue( len(resp.data) > 0 ) 
+        self.assertTrue(data[0]['name'] == 'TV')
+        self.assertTrue(data[1]['name'] == 'Blender')
+
+    def test_get_product_price(self):
+        resp1 = self.app.get('/products?min-price=100')
```

Comment:
Perhaps it's a personal preference, but I find naming variables based upon what they intend to store/test - to be clearer than incrementing the variable name.

So instead of 'resp1', 'resp2' and 'data1', 'data2' you could use:
'resp_min_price', 'resp_max_price' and 'data_min_price', 'data_max_price' - respectively.

Again, this is subjective, so I leave it to your discretion. My $0.02.